python/cac_tripplanner/templates/home.html

```diff
@@ -320,14 +320,6 @@
     subdomains: 'abcd',
     maxZoom: 19
     }).addTo(map);
-
-    // set class on body
-    // FIXME: avoid having to do this for templating to work.
-    // also had to wrap contents here in a div with the same class
-    // leave out class on div or class on body, and either have no map,
-    // or map and no content overlay
-    $('body').removeClass();
-    $('body').addClass('body-home');
```

Comment:
This gets rid of this class manipulation but still leaves the new div.  Is the plan to use the `body-home` and `body-map` classes only on the div, not on `<body>`?